src/main/java/com/gtnewhorizon/structurelib/util/EventHandler.java

```diff
@@ -0,0 +1,45 @@
+package com.gtnewhorizon.structurelib.util;
+
+import com.gtnewhorizon.structurelib.StructureLib;
+import com.gtnewhorizon.structurelib.item.ItemDebugStructureWriter;
+import com.gtnewhorizon.structurelib.item.ItemDebugStructureWriter.Mode;
+import com.gtnewhorizon.structurelib.net.UpdateDebugWriterModePacket;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.client.event.MouseEvent;
+
+public class EventHandler {
+    @SideOnly(Side.CLIENT)
```

Comment:
If this is a client only event handler, please put it in the event handlers registered by ClientProxy, not by using SideOnly.

Reply:
I did it like this in case there were more event handling functions added in the future

Reply:
`@SideOnly` is not intended to be used by mod devs, you should split the class.